cpp/augment_bible_references.cc

```diff
@@ -396,28 +382,26 @@ bool FindGndCodes(const bool verbose, const std::string &tags, const MarcRecord
 
     bool found_at_least_one(false);
     for (const auto &tag : individual_tags) {
-        for (size_t index(record.getFieldIndex(tag));
-             index < record.getNumberOfFields() and record.getTag(index) == tag; ++index)
-        {
-            const Subfields subfields(record.getSubfields(index));
-            const std::string subfield2(subfields.getFirstSubfieldValue('2'));
+        const auto field_range(record.getTagRange(tag));
```

Comment:
If you con't use field_range for anything but the loop below, please eliminate it!